sql/moz-fx-data-shared-prod/fxci_derived/worker_costs_v1/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+friendly_name: Firefox-CI Worker Costs
+description: |-
+  Derived Firefox-CI worker cost data from the GCP billing export.
+owners:
+- ahalberstadt@mozilla.com
+labels:
+  incremental: true
+  owner1: ahalberstadt
+scheduling:
+  dag_name: bqetl_internal_tooling
+bigquery:
+  time_partitioning:
+    type: day
+    field: ''
```

Comment:
I triggered integration tests and this empty field causes deployment to fail: https://app.circleci.com/pipelines/github/mozilla/bigquery-etl/35791/workflows/a29ff4dc-f2b8-4b1e-a72d-ba787e14e0a5/jobs/401942

There doesn't seem to be any date/time field here. Is this intentional? It generally makes sense to capture aggregates over a specific timeframe.

Reply:
Hm.. It's somewhat intentional?

This table is meant to provide the total cost of a GCE instance as used in Taskcluster. Instances are typically only alive for a few hours to a few days, so querying their cost over time doesn't make much sense.

 That said, we will want to limit results based on date so there's absolutely a field missing. I don't really want to group by `usage_start_time` as that has hourly granularity which I don't think anyone will care about. But grouping by day should be good enough.